networking/hub-default.json

```diff
@@ -70,7 +70,7 @@
             "[concat(variables('baseFwPipName'), '-02')]"
         ],
         "hubFwName": "[concat('fw-', parameters('location'))]",
-        "fwPoliciesBaseName": "fw-policies-base",
+        "fwPoliciesBaseName": "[concat('fw-policies-base-', parameters('location'))]", 
```

Comment:
While policies are regional in where they need to be defined, it is my understanding that they can (and are designed to be) applied globally (meaning, to a FW in any region).  Base policies are likely non-regional in nature because they are designed to be applied globally at the enterprise.  What problem does adding region solve to the base policy?

Reply:
deleted.  It is without the region in the name, now.